First steps

prosenfeld opened the pull request.

computerjoe314 committed:
First steps

isoboroff commented on evalbase/evalbase/models.py:
```diff
@@ -38,7 +38,7 @@ class Conference(models.Model):
     complete = models.BooleanField()
     agreements = models.ManyToManyField('Agreement', blank=True)
-    results_root = models.CharField(
+    results_root: str = models.CharField(
```
I don't want to add type annotations unless the whole codebase gets the treatment.  I also haven't coded in Python with type annotations before so this might be a big change to do.

Suggest dropping this change.

isoboroff commented on evalbase/evalbase/settings.py:
```diff
@@ -134,7 +134,10 @@
 
 # Email setup
-EMAIL_HOST = 'smtp.nist.gov'
+# EMAIL_HOST = 'smtp.nist.gov'
```
This should be in a local branch on your side and not pushed upstream

isoboroff commented on evalbase/templates/evalbase/home.html:
```diff
@@ -34,14 +34,23 @@
               ({{ conf.shortname }})</a>
             {{ conf.longname }}</li>
-          {% endfor %}
+          {% endfor %}s
```
Typo

Commit:
More features!
- Download uploaded files (Not working in admin)
- Edit previous submissions. Works but forces reupload on file.
- Updated some dependencies.

Commit:
Fix settings.py for upstream as requested

isoboroff commented on evalbase/evalbase/forms.py:
```diff
@@ -64,6 +66,70 @@ def get_form_class(context):
                                                            choices=[('yes', 'Yes'), ('no', 'No')])
         return type('SubmitFormForm', (forms.Form,), fields)
 
+    def edit_form_class(context):
```
Nice!

isoboroff reviewed (approved):
Nice edits.  I like the update with the edit-submission form.
